### PR TITLE
Feature/v3 hms test a new dawn

### PR DESCRIPTION
**Pull request by nieuwsma**
### Summary and Scope

massive rework to how the hms-tests are built and executed. Creates a framework for executing the CT tests.

### Testing
created a comprehensive runIntegration.sh

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,45 @@ Removed - for now removed features
 Security - in case of vulnerabilities
 -->
 
+## [Unreleased] - 2022-02-17
```
Comment (nieuwsma) [inline CHANGELOG.md:27]: need to clean up changelog!

**Comment by nieuwsma** [comment]
I need to write some general documentation

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
 # HMS Continuous Test (CT) Base Infrastructure Repository
 
-This repository contains base infrastructure (RPM build, shared libraries, utilities, etc) required to deploy and execute the HMS CT tests that live in other HMS repositories along with the services that they verify. The hms-ct-test-base.spec file generates hms-ct-test-base RPMs which are installed on Shasta system NCNs and provide infrastructure expected by the HMS CT tests during runtime. The hms-ct-test-base RPM by itself does not include tests for HMS services.
+This is a MAJOR redisign (v3); go see v1 code for how the RPMs were created.
+This repository contains the docker image `hms-test` that is inherited by the `continious test` images (eg: `cray-fimrware-action-test` in `hms-firmware-action` repo.
```
Comment (schooler-hpe) [inline README.md:4]: Typo here on `continuous`.

File: cmd/smoke_test.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+    logging.debug("default_url: %s", default_url)
+    logging.debug("override_url: %s", override_url)
+    logging.debug("configured_url: %s", configured_url)
+
+    signal.signal(signal.SIGINT, signal_handler)
+    signal.signal(signal.SIGHUP, signal_handler)
+    signal.signal(signal.SIGTERM, signal_handler)
+
+    ##TEST CODE######os.kill(os.getpid(), signal.SIGHUP)
```
Comment (schooler-hpe) [inline cmd/smoke_test.py:96]: This can be deleted.

File: docker-compose.integration.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+version: '3.7'
+#THIS environment sets up all of FAS
```
Comment (schooler-hpe) [inline docker-compose.integration.yaml:2]: This comment isn't relevant, looks like the result of a copy/paste.

File: runIntegration.sh [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+function cleanup() {
+  docker-compose down
+  if ! [[ $? -eq 0 ]]; then
+    echo "Failed to decompose environment!"
+    exit 1
+  fi
+  exit $1
+}
+
+# Step 3) Get the base containers running
```
Comment (schooler-hpe) [inline runIntegration.sh:46]: "Step 3)" must be a copy/paste thing that's not needed/true here.

**Review by schooler-hpe — APPROVED** [review]

**Review by rsjostrand-hpe — APPROVED** [review]

**Pull request merged**